Makefile

```diff
@@ -0,0 +1,2 @@
+osgincommoncertrequest.html: man/osg-incommon-cert-request.1
```

Comment:
I prefer for the .html file to be in the man directory.  It would probably be a good idea to put a comment in the Makefile too to say that this is for links to the web page via http://htmlpreview.github.io, and that whenever the man page source changes, 'make' should be run and the resultant .html file checked in to github.

Reply:
Done

Reply:
The make destination also should be in the man directory, to prevent make from doing anything if the file hasn't changed.

Reply:
Done